providers/execute.rb

```diff
@@ -5,11 +5,16 @@
 # Author:: Jamie Winsor (<jamie@vialstudios.com>)
 #
 
+require 'etc'
+
 include Chef::Mixin::Rbenv
 
 def load_current_resource
+  @user                         = new_resource.user || Etc.getlogin
+  @home                         = Etc.getpwnam(@user).dir
   @path                         = [ rbenv_shims_path, rbenv_bin_path ] + new_resource.path + system_path
   @environment                  = new_resource.environment
+  @environment["HOME"]          = @home
```

Comment:
Hm, good point. I've been trying to properly fix it, and future-proof it a bit, but it's probably better to just fix the immediate issue.